app/src/main/java/com/bunbeauty/papakarlo/feature/cafe/ui/CafeStatusColor.kt

```diff
@@ -2,14 +2,41 @@ package com.bunbeauty.papakarlo.feature.cafe.ui
 
 import androidx.compose.runtime.Composable
 import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.platform.LocalConfiguration
+import androidx.compose.ui.res.stringResource
+import com.bunbeauty.papakarlo.R
 import com.bunbeauty.papakarlo.common.ui.theme.FoodDeliveryTheme
-import com.bunbeauty.shared.domain.model.cafe.CafeStatus
+import com.bunbeauty.shared.presentation.cafe_list.CafeItem
 
 @Composable
-fun getCafeStatusColor(cafeStatus: CafeStatus): Color {
+fun getCafeStatusColor(cafeStatus: CafeItem.CafeOpenState): Color {
     return when (cafeStatus) {
-        CafeStatus.OPEN -> FoodDeliveryTheme.colors.statusColors.positive
-        CafeStatus.CLOSE_SOON -> FoodDeliveryTheme.colors.statusColors.warning
-        CafeStatus.CLOSED -> FoodDeliveryTheme.colors.statusColors.negative
+        is CafeItem.CafeOpenState.Opened -> FoodDeliveryTheme.colors.statusColors.positive
+        is CafeItem.CafeOpenState.CloseSoon -> FoodDeliveryTheme.colors.statusColors.warning
+        is CafeItem.CafeOpenState.Closed -> FoodDeliveryTheme.colors.statusColors.negative
     }
 }
+
+@Composable
+fun getCafeStatusText(cafeStatus: CafeItem.CafeOpenState): String {
```

Comment:
Move this logic to mapper. CafeStatusColor is not the right place for this.

Reply:
hmmm
mb baby
but here i have working with resources